install.sh

```diff
@@ -167,7 +168,8 @@ function install() {
         rm -rf "dist/" 2> "/dev/null" || true
     fi
 
-    cp -n "./config.json" "$DEST_DIR$SYSCONF_DIR/fw-fanctrl" 2> "/dev/null" || true
+    cp -pn "./src/fw_fanctrl/_resources/config.json" "$DEST_DIR$SYSCONF_DIR/fw-fanctrl" 2> "/dev/null" || true
+    cp -f "./src/fw_fanctrl/_resources/config.schema.json" "$DEST_DIR$SYSCONF_DIR/fw-fanctrl" 2> "/dev/null" || true
```

Comment:
If I understand this code correctly, it would install the JSON schema in the configuration directory? Is this something a user wants/needs to edit?